### ISSUE TITLE
Lets follow a mobile-first design approach

### ISSUE THREAD

**Issue by prampey** [OSDLabs/swd#34]
One of the key reasons for redesigning SWD was so that it could provide a pleasant _mobile experience_. 
We should design components with the mobile layout in mind **first**. This is simpler to achieve and scaling to desktop screens becomes more intuitive, rather than trying to fit desktop designs into mobile screen estate.

**Comment by UtkarshMe** [comment]
Should we consider [Progressive Web Apps](https://developers.google.com/web/progressive-web-apps/)? I like to use them very much. Mobile app won't be needed.

**Comment by prampey** [comment]
Yes, this is the next step after we're done with a mobile UI and a working prototype.
